plugins/modules/api_info.py

```diff
@@ -0,0 +1,262 @@
+      cmdb_ci: ""
+      comments: ""
+      comments_and_work_notes: ""
+      company: 31bea3d53790200044e0bfc8bcbe5dec
+      contact_type: phone
+      contract: ""
+      correlation_display: ""
+      correlation_id: ""
+      delivery_plan: ""
+      delivery_task: ""
+      description: Noticing today that any time I send an email with an attachment, it
+        takes at least 20 seconds to send.
+      due_date: ""
+      escalation: "0"
+      expected_start: ""
+      follow_up: ""
+      group_list: ""
+      hold_reason: ""
+      impact: "1"
+      incident_state: "7"
+      knowledge: "false"
+      location: ""
+      made_sla: "false"
+      notify: "1"
+      number: INC0000013
+      opened_at: "2020-07-06 23:15:58"
+      opened_by: 9ee1b13dc6112271007f9d0efdb69cd0
+      order: ""
+      parent: ""
+      parent_incident: ""
+      priority: "1"
+      problem_id: ""
+      reassignment_count: "2"
+      reopen_count: ""
+      reopened_by: ""
+      reopened_time: ""
+      resolved_at: "2020-09-24 19:54:48"
+      resolved_by: 6816f79cc0a8016401c5a33be04be441
+      rfc: ""
+      route_reason: ""
+      service_offering: ""
+      severity: "3"
+      short_description: EMAIL is slow when an attachment is involved
+      sla_due: ""
+      state: "7"
+      subcategory: ""
+      sys_class_name: incident
+      sys_created_by: don.goodliffe
+      sys_created_on: "2020-07-07 23:18:07"
+      sys_domain: global
+      sys_domain_path: /
+      sys_id: 46cebb88a9fe198101aee93734f9768b
+      sys_mod_count: "5"
+      sys_tags: ""
+      sys_updated_by: VALUE_SPECIFIED_IN_NO_LOG_PARAMETER
+      sys_updated_on: "2020-09-24 19:54:48"
+      task_effective_number: INC0000013
+      time_worked: ""
+      universal_request: ""
+      upon_approval: ""
+      upon_reject: ""
+      urgency: "1"
+      user_input: ""
+      watch_list: ""
+      work_end: ""
+      work_notes: ""
+      work_notes_list: ""
+      work_start: ""
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+from ..module_utils import arguments, client, errors, table, utils
+from ..module_utils.api import (
+    transform_query_to_servicenow_query,
+    POSSIBLE_FILTER_PARAMETERS,
+    table_name,
+    FIELD_COLUMNS_NAME,
+)
+
+
+def run(module, table_client):
+    search_dict = dict(module.params)
+    columns = ",".join([field.lower() for field in module.params[FIELD_COLUMNS_NAME]])
+    search_dict.update(columns=columns)
+    query = utils.filter_dict(search_dict, *POSSIBLE_FILTER_PARAMETERS)
+    servicenow_query = transform_query_to_servicenow_query(query)
+    return table_client.list_records(table_name(module), servicenow_query)
+
+
+def main():
+    arg_spec = dict(
+        arguments.get_spec("instance", "sys_id"),
+        resource=dict(type="str", required=True),
+        sysparm_query=dict(type="str"),
+        display_value=dict(
+            type="str", choices=["true", "false", "both"]
```

Comment:
```suggestion
            type="str", choices=["true", "false", "all"], default="false"
```

Reply:
When set to `both`, the output is not correct, because the value that ServiceNow accepts is `all`.

Reply:
Thanks for noticing. Resolved with https://github.com/ansible-collections/servicenow.itsm/pull/179/commits/f25500401bfc45b275f18fd1f7a808c850554fe8.